src/core/main.c

```diff
@@ -698,6 +698,19 @@ static int parse_config_file(void) {
 
         config_parse_many(fn, conf_dirs_nulstr, "Manager\0", config_item_table_lookup, items, false, NULL);
 
+        _cleanup_free_ char *rvalue = NULL;
+        if (get_proc_cmdline_key("systemd.default_timeout_start_sec=", &rvalue)) {
+                usec_t jobtimeout;
+                int r;
+
+                r = parse_time(rvalue, &jobtimeout, 1);
```

Comment:
Hmm? parse_time() with the last parameter of 1 uses 1us as default time unit if none is specified. That should be 1s however, hence use USEC_PER_SEC as paramater there. Or just use parse_sec() instead, which is equivalent.